trio/_core/_tests/test_multierror.py

```diff
@@ -484,9 +486,9 @@ def test_pickle_multierror(protocol: int) -> None:
     my_except = ZeroDivisionError()
 
     try:
-        1 / 0
-    except ZeroDivisionError as e:
-        my_except = e
+        1 / 0  # noqa: B018  # Need exception to fire
+    except ZeroDivisionError as exc:
+        my_except = exc
```

Comment:
*very* minor nit, but I think it's quite obvious why the statement is there, and what needs a comment is why the `noqa` is needed - which is once again `"useless statement"` - to not make anybody think that what's being complained about is the division by zero.

Reply:
Maybe just replace it with `raise ZeroDivisionError`?

Reply:
Changed noqa additional comment details in a5afed7